src/views/CronTabList/CronTabList.tsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { CronTabModel } from '@crontab-model/CronTabModel';
+import { CronTabKind } from '@crontab-model/CronTabModel';
+import { modelToGroupVersionKind, modelToRef } from '@crontab-utils/utils';
+import {
+  K8sResourceCommon,
+  ListPageBody,
+  ListPageCreate,
+  ListPageFilter,
+  ListPageHeader,
+  Timestamp,
+  useK8sWatchResource,
+  useListPageFilter,
+  VirtualizedTable,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { useCronTabTranslation } from '@crontab-utils/hooks/useCronTabTranslation';
+import { ResourceLink, RowProps, TableData } from '@openshift-console/dynamic-plugin-sdk';
+import { TableColumn } from '@openshift-console/dynamic-plugin-sdk';
+import { sortable } from '@patternfly/react-table';
+
+import CronTabRowActions from './CronTabRowActions';
+
+
+type CronTabListProps = {
+  namespace: string;
+};
+
+const CronTabList: React.FC<CronTabListProps> = ({ namespace }) => {
+  const [cronTabs, loaded, loadError] = useK8sWatchResource<K8sResourceCommon[]>({
+    isList: true,
+    groupVersionKind: {
+      group: 'stable.example.com',
+      kind: 'CronTab',
+      version: 'v1',
+    },
+    namespaced: true,
+    namespace,
+  });
+  const { t } = useCronTabTranslation();
+  const columns = useCronTabColumns();
+  const [data, filteredData, onFilterChange] = useListPageFilter(cronTabs);
+
+  return (
+    <>
+      <ListPageHeader title={t('CronTab')}>
+        <ListPageCreate groupVersionKind={modelToRef(CronTabModel)}>Create CronTab</ListPageCreate>
```

Comment:
Should `groupVersionKind` be the object, not the reference string? If that's not the case, we need to open a bug against the SDK.

Reply:
Unfortunately `groupVersionKind` is type of [GroupVersionKind](https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts#L177) which is string.
We need to update to K8sGroupVersionKind which is an object

Reply:
Created https://issues.redhat.com/browse/OCPBUGS-13808